src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -108,7 +119,8 @@ export function Tab({
       aria-label={accessibilityLabel}
       onMouseUp={handleMouseUpByBlurring}
     >
-      <span className={styles.Title}>{children}</span>
+      <span className={tabTitleClassNames}>{children}</span>
+      {newDesignLanguage && <div className={selectedUnderlineClassName} />}
```

Comment:
This should be a `span` because a `div` is a block element and a button is inline-block. Have you tried simply using a `before` selector? It seems like a good candidate for this and reduces the mark-up necessary.

Reply:
+1 to using a `before` pseudo element for this, as it's presentational in nature